### PR TITLE
Feature/touch controls

### PR DESCRIPTION
**Pull request by pfiaux**
Adds basic buttons (arrows, space, escape), and a way to toggle them. Off by default.

- [x] Keep top buttons line smaller the canvas
- [x] Improve name/alignment of the escape and space keys

Solves https://github.com/neophob/ganother-world/issues/33 (well if it runs on mobile that is)

### CODE REVIEW

**Comment by neophob** [comment]
very nice, i like it! some minor notes:
- text above the canvas is too long (lopnger than the canvas) - you could fiddle around with the badge css thing so the text fits.
- i discovered the keyboard icon only after zooming in ;)
- the ESC button looks not centered vertically
- the Sp looks weird ... and its more FIRE than Space

File: wasm/main.js [hunk 1]
```diff
@@ -87,6 +91,45 @@
...
+
+    const toggleButton = document.getElementById('toggle-keyboard');
+    toggleButton.addEventListener('click', (e) => {
+      toggleKeyboard(toggleButton);
+    });
+  }
+
+  function toggleKeyboard(button) {
+    showKeyboard = !showKeyboard;
+    const changeToLabel = showKeyboard ? '⌨︎ ON' : '⌨︎ OFF';
```
Comment (neophob) [inline wasm/main.js:125]: can we show just the keyboard?

Comment (pfiaux) [reply wasm/main.js:125]: We could might be hard to understand. Maybe there's an eye open/closed glyph i can use next to it?

File: wasm/index.html [hunk 2]
```diff
@@ -9,33 +9,64 @@
...
         padding: .25em .3em;
         line-height: 1;
         border-radius: .25rem;
         font-weight: 700;
         text-decoration: none;
       }
-      #gotherworld {
+      .badge-invert {
+        color: #111;
+        background-color: #eee;
```
Comment (pfiaux) [inline wasm/index.html:28]: we could use the badge-invert to show which part and debug level is selected as well.

**Comment by pfiaux** [comment]
>     * the ESC button looks not centered vertically
I'm using [unicode](https://en.wikipedia.org/wiki/Unicode_control_characters) to display them so it's not 'E' 'S' 'C' but '␛' as 1 glyph. It seems to be a bit subscript like it's bottom aligned but smaller than regular fonts that's why.

Could use ESC instead.
 
>     * the Sp looks weird ... and its more FIRE than Space

I can rename it to FIRE, I use space because that's the mapping but yea not very user friendly.

File: wasm/index.html [hunk 3]
```diff
@@ -65,9 +101,17 @@ <h1>
...
     </p>
-    <p>
+    <div id="gotherworld-container">
+      <div id="touch-controls" class="hide">
+        <button alt="up key input" id="key-up">↑</button>
+        <button alt="left key input" id="key-left">←</button>
+        <button alt="down key input" id="key-down">↓</button>
+        <button alt="right key input" id="key-right">→</button>
+        <button alt="space key input" id="key-sp" class="align-right">ESC</button>
+        <button alt="escape key input" id="key-esc" class="align-right">FIRE</button>
```
Comment (pfiaux) [inline wasm/index.html:111]: Changed these to plain text, what do you think?

**Comment by neophob** [comment]
nice job! only thing is that the FIRE and ESC are mixed up... i loaded level 2, moved the cage left and right, picked up the gun - then you see it

**Comment by pfiaux** [comment]
😱 oh that's bad. Fixed now!

**Comment by neophob** [comment]
h0000ray!

**Pull request merged**